grazer/src/main/java/edu/asu/diging/grazer/core/fileupload/service/impl/FileUploadServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package edu.asu.diging.grazer.core.fileupload.service.impl;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.OffsetDateTime;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.commons.io.FilenameUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+import edu.asu.diging.grazer.core.domain.impl.TransformationFilesMetadataImpl;
+import edu.asu.diging.grazer.core.fileupload.db.impl.FileMetadataDatabaseConnection;
+import edu.asu.diging.grazer.core.fileupload.service.IFileUploadService;
+
+@Service
+@PropertySource(value = "classpath:config.properties")
+public class FileUploadServiceImpl implements IFileUploadService {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private static final String DATE_PATTERN = "yyyyMMddhhmmss";
+    
+    @Autowired
+    private FileMetadataDatabaseConnection connection;
+
+    @Autowired
+    private Environment env;
+    
+    public void uploadFiles(CommonsMultipartFile[] multipartFile) {
```

Comment:
why is the parameter called ```multipartFile``` (singular) when its type is array?

Reply:
and why do you use an array when this will always be two files?

Reply:
As the code for uploading the files would be the same, instead of writing it for both the files, I could just use the for loop if it was in an array